privacyidea/lib/resolvers/PasswdIdResolver.py

```diff
@@ -269,7 +269,7 @@ def getUserId(self, LoginName):
         # We do not encode the LoginName anymore, as we are
         # storing unicode in nameDict now.
         if LoginName in self.nameDict:
-            return self.nameDict[LoginName]
+            return str(self.nameDict[LoginName])
```

Comment:
I think we have the same problem here with Python 2.7, we should use ``six.text_type``.

Reply:
Fixed in 61743ea